docs/src/Model_Reference/Resources/fusion.md

```diff
@@ -0,0 +1,289 @@
+# Fusion
+_Added in v0.4.2_
+
+Fusion power plants could be a future source of clean, firm electricity.
+Most proposals for fusion plants use a thermal cycle, where the energy of fusion heats a working fluid.
+The working fluid passes through a turbine connected to a generator.
+These plants would be broadly similar to fission plants.
+They might also have some fusion-specific operational behaviors, which are implemented in this module.
+
+This fusion module is implemented **only** for thermal plants with unit commitment (THERM=1).
```

Comment:
I think it will work but it will not give useful results. It will completely ignore the fusion-specific parts of the model.

I think there is a reasonable relaxation of fusion to a `THERM_NO_COMMIT` model: basically a fusion plant with its pulse behavior averaged out over time. This would be equivalent to a standard thermal plant + an extra load proportional to fusion capacity.

I could do this but I'm feeling lazy about it. And it would need some additional conditions on the back-end / outputs. Maybe let's chat about it?

Reply:
Maybe at the very least the user should get an error (even more than a warning) when running fusion with ucommit=2 or model = 2. An error is called for because the output would be different from what the modeler would reasonably expect.

Reply:
Yes, I agree with adding a check to ensure that a model with fusion resources has ucommit > 0 in the settings. If the user wants to run it without ucommit, they can simply remove the fusion plants. In principle, we could handle this internally, but it would be less explicit.